### ISSUE TITLE
PageTitle shows "accidently" scrollbar on PC but not on mobile

### ISSUE THREAD

**Issue by Shotte** [t3solution/t3sbootstrap#384]
Typo 12.4.11
T3SB 5.3.5

The PageTitle shows on PC a scrollbar, which shouldn't be, beacuse the content isn't too big for that (in my opinion). It announces "Flex" and "Scroll" in Firefox F12 developer view but I cannot find the reason for that, because the scrollbar cannot be seen on mobile devices, which is correct.

Here some picures for more detailled illustration:
![titlebar](https://github.com/t3solution/t3sbootstrap/assets/6080354/44bd60f6-448c-4e95-8f74-b713d840180a)
![diagnostic](https://github.com/t3solution/t3sbootstrap/assets/6080354/fab766af-259b-4032-907a-5daffebddfe9)
![mobile-view-short](https://github.com/t3solution/t3sbootstrap/assets/6080354/4bdce4de-68b5-428f-a53e-96397d9cc404)

**Comment by t3solution** [comment]
In the event that a drop-down menu is longer than the screen height, "overflow: scroll" must be set here so that all pages are accessible.

**Comment by t3solution** [comment]
This should work for you - **SETUP**:
```
page.cssInline {
    10 = TEXT
    10.value (
      @media (max-width: 575px) {
        .navbar-expand-no {
          overflow-y: unset !important;
        }
      }
    )
}
```

**Comment by Shotte** [comment]
@t3solution 
Thank you again for your help, but I have still no success.
- I added your code to TYPOSCRIPT section: no success
- I changed the code skipping the @media part: still no success
- I switches to "Custom Bootswatch SCSS Override" and added simply ".navbar-expand-no {overflow-y: unset !important;}": again no success

After that chage I checked the "<style>" section in the page header and I identified, that the ".navbar-expand-no {...}" occurred twice. First as I added to custom section and second with ".navbar-expand-no {overflow-y: scroll !important;}", which comes from t3sbootstrap.css.

Finally I tested using "bootstrap.config.disableDefaultCss=1" in TYPOSCRIPT constants section which succeeded but forced of course other ugly formating issues, because the default styles were noch longer loaded and I had just overwritten some of them.

However, it seems to me that I'm currently unable to overwrite this style except I patch your original code from t3sbootstrap.css. As you can see I'm again still witout any idea, but maybe you have some for me.

Please keep in mind that I use also these parameters for TYPO3 (compression and concatination):
config.no_cache = 0
config.concatenateCss = 1
config.concatenateJs = 1
config.compressCss = 1
config.compressJs = 1
config.moveJsFromHeaderToFooter = 1
config.includeJSFooter = 1

I haven't played around with these parameters yet but potetially are there some sideffects based on compression and concatination.

**Comment by t3solution** [comment]
You already know: The above setup only takes effect if the screen width is smaller than 575px!

**Comment by Shotte** [comment]
Yes, but only if I use your code "@media (max-width: 575px)", or am I wrong?

I changed it to simply overwrite: ".navbar-expand-no {overflow-y: unset !important;}" and nothing else which should in my opinion succeed.

**Comment by t3solution** [comment]
... is your webside online?
A link is helpfull!

**Comment by Shotte** [comment]
yes: https://www.greedy-plants.com/

**Comment by t3solution** [comment]
Why is the bootstrap CSS etc. loaded inline? That will be the problem!

**Comment by Shotte** [comment]
Aaaah, you are absolutely right! Thank you very much very much for this helpful hint. I was not aware about this sideeffect. I just followed some instructions from anywhere for including css via TYPOSCRIPT.

Now I added "cssfile.inline = 0" to tell Typo3 explicitly not using it inline and it works. For me is this issue closed now.

Thank you again for your time you spend on an issue that was based on my weak knowledge about Typo3.
